app/src/main/java/net/gsantner/opoc/util/TextUtils.java

```diff
@@ -325,30 +327,28 @@ public static void selectLines(final EditText edit, final List<Integer> position
         }
         final CharSequence text = edit.getText();
         if (positions.size() == 1) { // Case 1 index
-            final int posn = StringUtils.getIndexFromLineOffset(text, positions.get(0), 0);
-            showSelection(edit, posn, posn);
-            edit.setSelection(posn);
+            final int posn = TextUtils.getIndexFromLineOffset(text, positions.get(0), 0);
+            setSelectionAndShow(edit, posn);
         } else if (positions.size() > 1) {
             final TreeSet<Integer> pSet = new TreeSet<>(positions);
             final int selStart, selEnd;
             final int minLine = Collections.min(pSet), maxLine = Collections.max(pSet);
             if (maxLine - minLine == pSet.size() - 1) { // Case contiguous indices
-                selStart = StringUtils.getLineStart(text, StringUtils.getIndexFromLineOffset(text, minLine, 0));
-                selEnd = StringUtils.getIndexFromLineOffset(text, maxLine, 0);
+                selStart = TextUtils.getLineStart(text, TextUtils.getIndexFromLineOffset(text, minLine, 0));
+                selEnd = TextUtils.getIndexFromLineOffset(text, maxLine, 0);
             } else { // Case non-contiguous indices
                 final String[] lines = text.toString().split("\n");
                 final List<String> sel = new ArrayList<>(), unsel = new ArrayList<>();
                 for (int i = 0; i < lines.length; i++) {
                     (pSet.contains(i) ? sel : unsel).add(lines[i]);
                 }
                 sel.addAll(unsel);
-                final String newText = TextUtils.join("\n", sel);
+                final String newText = android.text.TextUtils.join("\n", sel);
                 edit.setText(newText);
                 selStart = 0;
-                selEnd = StringUtils.getIndexFromLineOffset(newText, positions.size() - 1, 0);
+                selEnd = TextUtils.getIndexFromLineOffset(newText, positions.size() - 1, 0);
```

Comment:
Changed the name from `StringUtils` -> `TextUtils` to better reflect what it contains.